docs/HowToGuides/FirstPullRequest.md

```diff
@@ -0,0 +1,168 @@
+# How to Submit Your First Pull Request
+
+So you've decided to contribute to the Swift toolchain, welcome!
+Maybe this is your first time contributing to an open source project, or maybe
+you are an experienced open source contributor who is excited about Swift, or
+maybe you are somewhere in-between. Regardless of your background, we are
+excited to have you contribute and improve the developer experience for Swift
+programmers all over the globe.
+:sparkles::child::student::woman_technologist::technologist::man_technologist::sparkles:
+
+This document provides a high-level overview of different parts of the
+contribution process.
+
+## How do I pick something to work on?
+
+In case you don't have something specific you'd like to work on, such as
+implementing something for a Swift Evolution pitch, you could start off by
+working on a bug labeled `StarterBug` on [Swift JIRA][]. If the bug hasn't
+been assigned to someone, check the comments in case someone has already
+started working on it. If not, feel free to assign it to yourself and start
+working on it!
+
+[Swift JIRA]: https://bugs.swift.org
```

Comment:
@varungandhi-apple Maybe we can throw a link for the filtered issues already so people reading this may not be so familiar with JIRA and will be one click away from the starter bug issues? Or hits to search like use JQL like `labels = StarterBug and status = Open and project = Swift`?

Reply:
Good idea, that would be useful.